scalding-parquet/src/test/scala/com/twitter/scalding/parquet/tuple/TypedParquetTupleTest.scala

```diff
@@ -18,8 +16,10 @@ class TypedParquetTupleTest extends WordSpec with Matchers with HadoopPlatformTe
       import com.twitter.scalding.parquet.tuple.TestValues._
       HadoopPlatformJobTest(new WriteToTypedParquetTupleJob(_), cluster)
         .arg("output", "output1")
-        .sink[SampleClassB](TypedParquet[SampleClassB, BReadSupport](Seq("output1"))) { _.toSet shouldBe values.toSet }
-        .run
+        .sink[SampleClassB](TypedParquet[SampleClassB](Seq("output1"),
+          Macros.caseClassParquetReadSupport[SampleClassB](SampleClassB.schema))) {
```

Comment:
this should be an implicit argument and users should do something like:

```scala
import Macros.caseClassParquetReadSupport
```
in any file they use the implicit. If they want to be explicit, that's fine, implicit parameters can be passed explicitly.